front/pages/api/w/[wId]/use/chats/[cId]/index.ts

```diff
@@ -49,14 +49,13 @@ const chatMessageSchema: JSONSchemaType<ChatMessageType> = {
   type: "object",
   properties: {
     role: { type: "string" },
-    runRetrieval: { type: "boolean", nullable: true },
-    runAssistant: { type: "boolean", nullable: true },
     message: { type: "string", nullable: true },
     retrievals: {
       type: "array",
       items: chatRetrievedDocumentSchema,
       nullable: true,
     },
+    query: { type: "string", nullable: true },
```

Comment:
quick q: can that cause db sync issues?
(at the time the field is always unset when data is persisted)

Reply:
How would this be related to the db sync? I believe this is the schema for validating the incoming message from the client to the API endpoint?